acceptable/__main__.py

```diff
@@ -53,6 +54,12 @@ def parse_args(raw_args=None, parser_cls=None, stdin=None):
     metadata_parser.add_argument('modules', nargs='+')
     metadata_parser.set_defaults(func=metadata_cmd)
 
+    django_parser = subparser.add_parser(
+        'django-metadata',
+        help='Import django project and print extracted metadata in json')
+    django_parser.add_argument('django_app', nargs='*')
```

Comment:
You have 'project' in the help and 'app' in the argument name.

Reply:
I don't need that arg anyway, will remove